src/app/search/query.js

```diff
@@ -25,54 +26,70 @@ const addFieldQuery = (queryString, field = 'keyword') => {
    * Strip punctuation and process spaces as plus signs for final split.
    * ES characters to escape before sending: + - = && || > < ! ( ) { } [ ] ^ " ~ * ? : \ /
    */
-  const queryArr = queryString.replace(/[=(&&)(||)><!(){}\[\]^"~\*\?:\/-]/g, '\$&').trim().replace(/\s+/g, '+').split('+');
+  /**
+  const queryArr = queryString.replace(/[=(&&)(||)><!(){}\[\]^"~\*\?:\/-]/g, '\$&').trim()
+  .replace(/\s+/g, '+').split('+');
 
   // TODO: add an additional check on empty queries after the terms are processed.
 
   fieldQuery.push({ field, value: queryArr.join(' ') });
 
   return fieldQuery;
 };
+ */
+
+
+/**
+   * Strip punctuation and process spaces as plus signs for final split.
+   * ES characters to escape before sending: + - = && || > < ! ( ) { } [ ] ^ " ~ * ? : \ /
+ * @param {string} queryString
+ *
+ * @returns {string}
+   */
+const parseQuery = (queryString) => {
+  const queryArr = queryString.replace(/[=(&&)(||)><!(){}\[\]^"~\*\?:\/-]/g, '\$&')
+    .trim()
+    .replace(/\s+/g, '+')
+    .split('+')
+    .join(' ');
+  return queryArr;
+};
 
 /**
  * Uses a special format as provided by the ResearchNow Search API.
  * Format with possible values:
- * {
- *   queries:
- *   [{
- *        field: 'keyword',
- *        value: 'branch'
- *   }],
- *   filters:
- *   [{
- *        field: 'language',
- *        value: 'fr'
- *   }],
- *   sort:
- *   [{
- *        field: 'title.keyword',
- *        dir: 'asc'
- *   }],
- *   aggregations:
- *   [{
- *        type: 'terms',
- *        field: 'title.keyword'
- *   }],
- *   per_page: 10,
- *   page: 0
- * }
- *
+{
```

Comment:
These are doc strings so you need an `*` before each line.

Reply:
changed